Sources/MongoSwift/ChangeStream.swift

```diff
@@ -0,0 +1,155 @@
+import mongoc
+/// A token used for manually resuming a change stream. Pass this to the `resumeAfter` or `startAfter` fields of
+/// `ChangeStreamOptions` to resume or start a change stream where a previous one left off.
+/// - SeeAlso: https://docs.mongodb.com/manual/changeStreams/#resume-a-change-stream
+public struct ChangeStreamToken: Codable {
+    private let resumeToken: Document
+
+    internal init(resumeToken: Document) {
+      self.resumeToken = resumeToken
```

Comment:
indentation looks off here.
is there maybe some setting you can enable in VSCode to deal with these indentation issues?
(if you can't find one maybe ask @mbroadst since he uses it)
```suggestion
        self.resumeToken = resumeToken
```

Reply:
oops, yeah 😓. I think I was trusting vscode every time I hit enter and didn't realize it wasn't 4 spaces. I just got an extension called `indent one space` and another one `indenticator`  so I think that should help now.